### PR TITLE
Switched biglake database to use parent field implementation

### PR DESCRIPTION
**Pull request by melinath**
Switched to parent field implementation. This is safe to change since the resource was only [merged yesterday](https://github.com/GoogleCloudPlatform/magic-modules/pull/8753).

```release-note:none

```

### CODE REVIEW

File: mmv1/products/biglake/Database.yaml [hunk 1]
```diff
@@ -20,41 +20,36 @@ references: !ruby/object:Api::Resource::ReferenceLinks
   guides:
     "Manage open source metadata with BigLake Metastore": "https://cloud.google.com/bigquery/docs/manage-open-source-metadata#create_databases"
   api: "https://cloud.google.com/bigquery/docs/reference/biglake/rest/v1/projects.locations.catalogs.databases"
-base_url: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}/databases"
-self_link: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}/databases/{{name}}"
-create_url: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}/databases?databaseId={{name}}"
+base_url: "{{catalog}}/databases"
```
Comment (slevenick) [inline mmv1/products/biglake/Database.yaml:23]: Does this need to have a `%`? I thought we used % for parameters that can contain slashes

File: mmv1/templates/terraform/custom_import/biglake_database.go.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+    config := meta.(*transport_tpg.Config)
```
Comment (slevenick) [inline mmv1/templates/terraform/custom_import/biglake_database.go.erb:1]: We may not need a custom import if we use {{% in the import & ID format?

See resources like: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/healthcare/ConsentStore.yaml

File: mmv1/products/biglake/Database.yaml (hunk 1 above)
Comment (slevenick) [reply mmv1/products/biglake/Database.yaml:23]: Nevermind, that should be in the import format and id format instead of the URL

**Review by slevenick — CHANGES_REQUESTED** [review]

File: mmv1/templates/terraform/custom_import/biglake_database.go.erb (hunk 2 above)
Comment (melinath) [reply mmv1/templates/terraform/custom_import/biglake_database.go.erb]: Done & fixed a couple other issues with the original PR as well.

**Review by slevenick — APPROVED** [review]

**Pull request merged**